api/controllers/user.go

```diff
@@ -185,6 +187,38 @@ func HandleNewCodeValidation(c *gin.Context) {
 	c.IndentedJSON(http.StatusOK, gin.H{"message": "New Code created and sended"})
 }
 
+// controller to obtain the loomies from an user
+func HandleGetLoomies(c *gin.Context) {
+	userid, _ := c.Get("userid")
+
+	user, err := models.GetUserById(userid.(string))
+
+	// user exists or not
+	if err != nil {
+		if err == mongo.ErrNoDocuments {
+			c.AbortWithStatusJSON(http.StatusNotFound, gin.H{"message": "User was not found"})
+			return
+		} else {
+			c.AbortWithStatusJSON(http.StatusInternalServerError, gin.H{"message": "Internal server error"})
```

Comment:
I believe the format we're going for is:

```
{
	"error": boolean,
	"message": string
}
```

I think @PedroChaparro can shed some light into this.

Reply:
Yes, that's correct. The idea is to make all the endpoints to respond with the same format, at this way the client can trust that if there is some error (`"error": true`), there should be also a message to show using an alert or something like that (`"message": "The message"`).

Reply:
Sure, this PR was also open, I will change this in it as well! Thanks!